ToDoListApp/Todo.swift

```diff
@@ -10,20 +10,20 @@ import Foundation
 //class to represent a task.
 class Task {
     var title: String
-    var dueDate: Date
+    var category: String
     
     
-    init(title: String, dueDate: Date) {
+    init(title: String, category: String) {
         self.title = title
-        self.dueDate = dueDate
+        self.category = category
     }
  
 }
 
 
 var tasks: [Task] = [
-    Task(title: "Go to supermarket", dueDate: .now ),
-    Task(title: "Pay bills", dueDate: .now),
-    Task(title: "Make biscuits", dueDate: .now),
+    Task(title: "Go to supermarket", category: "Shopping"),
```

Comment:
Create a enum for all the category you will have 
```suggestion
  enum TaskCategory: String {
    case shopping = "Shopping"
    case finance = "Finance"
}
```